docs/source/workflows/design_workflows.rst

```diff
@@ -182,6 +187,10 @@ A branch has a name, along with any number of design workflows. A DesignWorkflow
 
 When you're done with a branch, it can be archived, removing it from the results of ``list`` and setting the ``archived`` flag.
 ``list_archived`` lists all archived branches in a project. An archived branch can be restored via its unique ID.
+Note that archiving branches are independent of archiving the design workflows contained within it.
+Archiving a branch, by removing it from listing methods, will hide the entire branch from default displays in the web UI.
+Similarly, archiving a design workflow will result in its executions and relevant assets no longer being displayed within the branch.
+Yet archiving a branch will NOT change the archived status of the contained design workflows in the context of design workflow listing methods.
```

Comment:
You can italicize using asterisks, which I think we should prefer over all caps for emphasis.